examples/yaml/service-common/avs/tls-aerospike-vector-search.yaml

```diff
@@ -0,0 +1,81 @@
+cluster:
+  cluster-name: "avs-db-1"
+feature-key-file: "/etc/aerospike-vector-search/secrets/features.conf"
+service:
+  metadata-namespace: "avs-meta"
+  ports:
+    5000:
+      tls-id: service-tls
+manage:
+  ports:
+    5040: { }
+
+heartbeat:
+  seeds:
+    - address: avs-app-aerospike-vector-search-0.avs-app-aerospike-vector-search-internal.avs.svc.cluster.local
+      port: 5001
+interconnect:
+  client-tls-id: interconnect-tls
+  ports:
+    5001:
+      addresses:
+        "0.0.0.0"
+      tls-id: interconnect-tls
+storage:
+  client-policy:
+    tls-id: aerospike-tls
+    credentials:
+      username: tester
+      password-file: "/etc/aerospike-vector-search/secrets/aerospike-password.txt"
+  seeds:
+    - aerocluster-0-0.aerocluster.aerospike.svc.cluster.local:
+        port: 4333
+        tls-name: "asd.aerospike.com"
+security:
+  auth-token:
+    private-key: "/etc/aerospike-vector-search/secrets/private_key.pem"
+    private-key-password: "/etc/aerospike-vector-search/secrets/client-password.txt"
+    public-key: "/etc/aerospike-vector-search/secrets/public_key.pem"
+tls:
+  service-tls:
+    trust-store:
+      store-file: /etc/ssl/certs/ca.aerospike.com.truststore.jks
+      store-password-file: /etc/ssl/certs/storepass
+      key-password-file: "/etc/ssl/certs/keypass"
+    key-store:
+      store-file: /etc/ssl/certs/svc.aerospike.com.keystore.jks
+      store-password-file: /etc/ssl/certs/storepass
+      key-password-file: /etc/ssl/certs/keypass
+#      override-tls-hostname: avs-app-aerospike-vector-search-0.avs-app-aerospike-vector-search.aerospike.svc.cluster.local
+
+  interconnect-tls:
+    trust-store:
+      store-file: /etc/ssl/certs/ca.aerospike.com.truststore.jks
+      store-password-file: /etc/ssl/certs/storepass
+      key-password-file: "/etc/ssl/certs/keypass"
+    key-store:
+      store-file: /etc/ssl/certs/avs.aerospike.com.keystore.jks
+      store-password-file: /etc/ssl/certs/storepass
+      key-password-file: /etc/ssl/certs/keypass
+    override-tls-hostname: avs.aerospike.com
+
+  aerospike-tls:
+    trust-store:
+      store-file: "/etc/ssl/certs/ca.aerospike.com.truststore.jks"
+      store-password-file: "/etc/ssl/certs/storepass"
+      key-password-file: "/etc/ssl/certs/keypass"
+    key-store:
+      store-file: "/etc/ssl/certs/avs.aerospike.com.keystore.jks"
+      store-password-file: "/etc/ssl/certs/storepass"
+      key-password-file: "/etc/ssl/certs/keypass"
+#      override-tls-hostname: "asd.aerospike.com"
+
+logging:
+  #    file: /var/log/aerospike-vector-search/aerospike-vector-search.log
```

Comment:
Should we include commented out configs?

Reply:
I think in an example it is fine.